.github/workflows/conformance.yml

```diff
@@ -67,7 +67,6 @@ jobs:
         - pubsub.hazelcast
         - pubsub.in-memory
         - pubsub.mqtt3-emqx
-        - pubsub.mqtt3-mosquitto
```

Comment:
Should Mosquitto be removed from `bindings.mqtt3-mosquitto` too?

Reply:
Bindings is different because we don't make the same guarantees there. Should be fine to leave in bindings IMO.